src/app/complex/complex-results/complex-navigator-buttons/complex-navigator-buttons.component.html

```diff
@@ -19,14 +30,14 @@
       <li>
         <label>
           <input type="radio" name="displayType" value="Compact" class="checkbox compact"
-                 (click)="setDisplayType('compact')" [checked]="typeOfDisplay === 'compact'">
+                 (click)="setDisplayType('compact')" [checked]="typeOfDisplay() === NavigatorDisplayType.COMPACT">
```

Comment:
Replace
```
setDisplayType('compact')
```

With 
```
setDisplayType(NavigatorDisplayType.COMPACT)
```

Same for line 40.

Reply:
We also need to update the type of the argument in `setDisplayType`
```
setDisplayType(type: NavigatorDisplayType) {
```

And update the ifs in that method to use the enum.